modules/k3d/README.md

```diff
@@ -0,0 +1,89 @@
+# k3d 
+
+https://k3d.io
+
+## Usage
+
+This example shows spinning up a K3d cluster, deploying a local image, and accessing the endpoint.
+
+Spin up the cluster:
+
+```
+make k3d/create
+```
+
+Validate that the nodes are up, and all pods are running:
+
+```
+kubectl get nodes
+kubectl get pods --all-namespaces
+```
+
+Create a local image, using [Nginx](https://www.nginx.com/) as an example:
+
+```
+# Tag it and push it to the docker-registry which this module creates for us
+
+docker tag nginx:latest k3d-default.localhost:5000/mynginx:v0.1           
+docker push k3d-default.localhost:5000/mynginx:v0.1               
```

Comment:
I had to add `127.0.0.01 k3d-default.localhost` to /etc/hosts to get this to work

Reply:
Ah, I have the following in mine (was there from the install)

```
::1	localhost ip6-localhost ip6-loopback
```

I wonder if this is why it works for me?

```
ping foo.localhost                              1 ↵
PING foo.localhost(localhost (::1)) 56 data bytes
64 bytes from localhost (::1): icmp_seq=1 ttl=64 time=0.038 ms
```